README.md

```diff
@@ -536,6 +536,12 @@ For versions of ISIS prior to ISIS 4.1.0, please cd into `$ISIS3DATA` instead of
 
 **New Horizons Mission:**
 
+    cd $ISISDATA
+    rsync -azv --delete --partial isisdist.astrogeology.usgs.gov::isisdata/data/osirisrex .
```

Comment:
These are flipped. The osirisrex download is under the "New Horizons Mission" header and vice-versa.